Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 #!/bin/groovy
-@Library('SonarSource@master') _
+@Library('SonarSource@1.2') _
```

Comment:
Should it stays to master or not? I remember it was 1 something then @drautureau-sonarsource changed it to master and now you change it back.

Reply:
@Evangelink changing to master was a temporary solution (waiting for the release 1.2)
Staying on a release version is recommended as your builds will be reproducible and you won't be impacted by refactoring tasks done on the library

Reply:
Make sense! Thx for the update